### PR TITLE
Dev docker

### PR DESCRIPTION
**Pull request by pyoner**

### CODE REVIEW

**Comment by pyoner** [comment]
@knopki
Please review the PR

File: README.md [hunk 1]
```diff
@@ -63,9 +63,9 @@ repositories cloned on the same level. Also, you need `docker` and `docker-compo
 Now, you can run `docker-compose up` (`podman-compose up`) in `metis-backend`
 directory. This should start all dependencies and services.
 
-`metis-gui` should be available at `http://localhost:10000/`
+`metis-gui` should be available at `http://localhost:5000/`
 
-`metis-bff` shoult be available at `http://localhost:10000/api`
+`metis-bff` shoult be available at `http://localhost:3000/api`
```
Comment (knopki) [inline README.md:68]: No, if you destroy nginx proxy at 10000 port, then `metis-bff`'s url is `http://localhost:3000`.

**Review by knopki — CHANGES_REQUESTED** [review]
Maybe I miss the point of this PR, but this doesn't work.

**Comment by knopki** [comment]
As I understand, you just want a comfortable dev environment. Please wait ~1h and I'll try to make it.

**Comment by knopki** [comment]
@pyoner plz check my version https://github.com/basf/metis-backend/pull/8

**Comment by blokhin** [comment]
Closing as advised by @knopki

**Pull request closed without merging**